index.html

```diff
@@ -68,6 +76,25 @@ <h4>Multi-Post Stories</h4>
           </ul>
           <button type="button">See Project</button>
         </div>
+        <div class="works-card-1-desktop">
+          <div class="works-card-1-image"><img src="./images/img_placeholder-desktop.png" alt="card-1-image"></div>
+          <div class="works-card-1-text">
+            <h4>Multi-Post Stories</h4>
+            <p class="card-content-1">A daily selection of privately personalized<br>
+            reads; no accounts or sign-ups<br>
+            has been the industry's standard dummy<br>
+            text ever since the 1500s, when an<br>
+            unknown printer took a standard dummy text.
+            </p>
+            <ul class="tech-stack">
+            <li>css</li>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+            </ul>
+            <button type="button">See Project</button>
+          </div>
```

Comment:
While you did a nice job working on the project requirements. There are still some issues you need to correct. The content of this part of the page seems to be overlaping. Please correct this and carefully inspect the figma template. 

This is your design:
![Screenshot from 2022-07-16 20-38-59](https://user-images.githubusercontent.com/79506105/179364363-30b396b1-e2a8-40bc-b848-9c127ae737ea.png)

Follow this design:
![Screenshot from 2022-07-16 20-40-47](https://user-images.githubusercontent.com/79506105/179364417-7d161a0f-37a7-4dea-94a1-2203357eb22b.png)